src/drive/web/modules/navigation/duck/actions.jsx

```diff
@@ -164,6 +164,27 @@ export const fetchMoreFiles = (folderId, skip, limit) => {
     }
   }
 }
+/**
+ *
```

Comment:
Can you add a description of the goal of the function ? I don't understand why it is related to the recent view.

Reply:
Should this method be part of the FileCollection ?

Reply:
Yes, and this is why this is not ported to CC yet. My goal was not to rewrite this part yet, since it uses this kind of code https://github.com/cozy/cozy-drive/blob/5f13f23069199c355385b5d68bd0ded42a4ee316/src/drive/web/modules/navigation/duck/actions.jsx#L203 based on  https://github.com/cozy/cozy-drive/blob/5f13f23069199c355385b5d68bd0ded42a4ee316/src/drive/web/modules/navigation/duck/async.js#L288 and it will be removed for the migration.

Reply:
OK, thanks for the explanation.